check_run/check_run/doctype/check_run/check_run.py

```diff
@@ -458,8 +462,11 @@ def confirm_print(docname):
 	# Remove PDF file(s)
 	remove_all("Check Run", docname, from_delete=False, delete_permanently=False)
 
-	# Reset status
-	return frappe.db.set_value("Check Run", docname, "status", "Printed")
+	# Reset status and enable hooks
+	frappe.db.set_value("Check Run", docname, "status", "Printed")
```

Comment:
Slight refactor here since we're calling `get_doc` anyway:
```
  # removed frappe.db.set_value("Check Run", docname, "status", "Printed")
	cr = frappe.get_doc("Check Run", docname)
  cr.db_set("status", "Printed")
	cr.run_method("on_update_after_submit")
```

This will prevent any timestamp mismatch errors

Reply:
No problem - latest changes inbound